config/sync/core.entity_view_display.node.article.default.yml

```diff
@@ -0,0 +1,67 @@
+uuid: dd960c84-ac23-488b-bb5c-1b9031573a45
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.article.field_a_body
+    - field.field.node.article.field_a_content_author
+    - field.field.node.article.field_a_hero_image
+    - field.field.node.article.field_a_summary
+    - field.field.node.article.field_a_teaser_image
```

Comment:
These fields are in the wrong order in the form display, they should be in the correct order as per task list

Reply:
done